scripts/ci/linux/build.sh

```diff
@@ -13,11 +13,18 @@ echo "PLATFORM_CFLAGS += $CUSTOMFLAGS" >> libs/openFrameworksCompiled/project/li
 cd libs/openFrameworksCompiled/project
 make Debug
 
-echo "Building emptyExample"
```

Comment:
building empty example makes the build crash and my suspicion is that this is happening because empty example is including ofMain.h which is huge. i've changed it to build the tests which are much lighter because they only include the headers that are needed and it seems to work better.

the tests are also normal applications so if those are building emptyExample should build too. 

empty example is also building in osx so we will catch most possible errors affecting emptyExample but not the tests there